include/igl/predicates/polygons_to_triangles.h

```diff
@@ -0,0 +1,52 @@
+// This file is part of libigl, a simple c++ geometry processing library.
+// 
+// Copyright (C) 2020 Alec Jacobson <alecjacobson@gmail.com>
+// 
+// This Source Code Form is subject to the terms of the Mozilla Public License 
+// v. 2.0. If a copy of the MPL was not distributed with this file, You can 
+// obtain one at http://mozilla.org/MPL/2.0/.
+#ifndef IGL_POLYGONS_TO_TRIANGLES_H
+#define IGL_POLYGONS_TO_TRIANGLES_H
+
+#include "../igl_inline.h"
+#include <Eigen/Core>
+#include <vector>
+
+namespace igl
+{
+  namespace predicates
+  {
+    // Given a polygon mesh, trivially triangulate each polygon with a fan. This
```

Comment:
Update comment? This function uses the ear clipping rather than the triangle fan approach it seems.